main.js

```diff
@@ -20,22 +20,25 @@ function sendErrorProd (err, req, res, next) {
 	if (err.name === fetchres.ReadTimeoutError.name) {
 		logger.error('event=dependencytimeout', err);
 		res && res.status(504).send({ type: 'Bad Gateway', error: err });
-	} else {
+	} else if (req && res && next) {
 		return ravenMiddleware(err, req, res, next);
+	} else {
+		logger.error('event=uncaughterror', err);
 	}
 }
 
 function getUpstreamErrorHandler (errorReporter) {
-	return function(res, statusCode) {
```

Comment:
this line introduced a bug (cf https://github.com/Financial-Times/next-stream-page/pull/731/files#diff-ca511433af8c6e7367fd73d55a2b0be1R272) which turned 404s on stream pages into 503s. Anyway, fixed now